src/main/scala/coreplex/RocketCore.scala

```diff
@@ -8,8 +8,8 @@ import uncore.coherence._
 import rocket._
 import uncore.devices.NTiles
 
-trait RocketPlex extends CoreplexRISCVPlatform {
-  val module: RocketPlexModule
+trait RocketCore extends CoreplexRISCVPlatform {
```

Comment:
I don't think core is the right word here.  Tiles contain cores, not the other way around.